plugins/inputs/ravendb/ravendb.go

```diff
@@ -0,0 +1,680 @@
+			"statistics_map_index_indexes_per_sec":        dbResponse.Statistics.MapIndexIndexesPerSec,
+			"statistics_map_reduce_index_mapped_per_sec":  dbResponse.Statistics.MapReduceIndexMappedPerSec,
+			"statistics_map_reduce_index_reduced_per_sec": dbResponse.Statistics.MapReduceIndexReducedPerSec,
+			"statistics_requests_per_sec":                 dbResponse.Statistics.RequestsPerSec,
+			"statistics_requests_count":                   dbResponse.Statistics.RequestsCount,
+			"statistics_request_average_duration":         dbResponse.Statistics.RequestAverageDuration,
+			"indexes_count":                               dbResponse.Indexes.Count,
+			"indexes_stale_count":                         dbResponse.Indexes.StaleCount,
+			"indexes_errors_count":                        dbResponse.Indexes.ErrorsCount,
+			"indexes_static_count":                        dbResponse.Indexes.StaticCount,
+			"indexes_auto_count":                          dbResponse.Indexes.AutoCount,
+			"indexes_idle_count":                          dbResponse.Indexes.IdleCount,
+			"indexes_disabled_count":                      dbResponse.Indexes.DisabledCount,
+			"indexes_errored_count":                       dbResponse.Indexes.ErroredCount,
+			"storage_documents_allocated_data_file_in_mb": dbResponse.Storage.DocumentsAllocatedDataFileInMb,
+			"storage_documents_used_data_file_in_mb":      dbResponse.Storage.DocumentsUsedDataFileInMb,
+			"storage_indexes_allocated_data_file_in_mb":   dbResponse.Storage.IndexesAllocatedDataFileInMb,
+			"storage_indexes_used_data_file_in_mb":        dbResponse.Storage.IndexesUsedDataFileInMb,
+			"storage_total_allocated_storage_file_in_mb":  dbResponse.Storage.TotalAllocatedStorageFileInMb,
+			"storage_total_free_space_in_mb":              dbResponse.Storage.TotalFreeSpaceInMb,
+		}
+
+		if dbResponse.TimeSinceLastBackupInSec != nil {
+			fields["time_since_last_backup_in_sec"] = dbResponse.TimeSinceLastBackupInSec
+		}
+
+		acc.AddFields("ravendb_databases", fields, tags)
+	}
+}
+
+func gatherIndexes(r *RavenDB, acc telegraf.Accumulator) {
+	if !r.GatherIndexStats {
+		return
+	}
+
+	indexesResponse := &indexesMetricResponse{}
+
+	err := r.requestJSON("/admin/monitoring/v1/indexes"+prepareDbNamesUrlPart(r.IndexStatsDbs), &indexesResponse)
+	if err != nil {
+		acc.AddError(err)
+		return
+	}
+
+	for _, indexResponse := range indexesResponse.Results {
+		tags := map[string]string{"url": r.URL}
+		tags["database_name"] = indexResponse.DatabaseName
+		tags["index_name"] = indexResponse.IndexName
+		tags["node_tag"] = indexesResponse.NodeTag
+
+		if indexesResponse.PublicServerUrl != "" {
+			tags["public_server_url"] = indexesResponse.PublicServerUrl
+		}
+
+		fields := map[string]interface{}{
+			"priority":        indexResponse.Priority,
+			"state":           indexResponse.State,
+			"errors":          indexResponse.Errors,
+			"lock_mode":       indexResponse.LockMode,
+			"is_invalid":      boolToInt(indexResponse.IsInvalid),
```

Comment:
We do understand bool... :-) Please keep it instead of converting to int.

Reply:
done

Reply:
anyway Grafana has issues with reading bools

Reply:
Well you can use a processor for your use-case to do the mapping you need, so we can have the best of two worlds.

Reply:
ok removed and I'm using boolean

Reply:
Please mark as resolved.